SparkPost/2web2ui
# SA-903: Release new navigation

### What Changed
 - Removed Feature flag from new navigation

### How To Test
 - Log in and see that new navigation categories exist
 - Check that links are working correctly.
 - Check on different user permissions (admin, developer, etc.) to see that different links are hidden correctly

### To Do
- [ ] Make the group links more accessible as a button (?)

### Notes
 - To roll out early Friday (9/20)
